optimum/onnx/configuration.py

```diff
@@ -339,6 +339,73 @@ def fill_with_past_key_values_(self, inputs_or_outputs: Mapping[str, Mapping[int
         for i in range(num_decoder_layers * num_pkv_per_layer):
             inputs_or_outputs[f"{name}_key_values_{i}"] = {0: "batch", 2: decoder_sequence}
 
+    def generate_dummy_inputs_onnxruntime(self, reference_model_inputs: Mapping[str, Any]) -> Mapping[str, Any]:
+        reference_model_inputs["encoder_attention_mask"] = reference_model_inputs.pop("attention_mask")
+        reference_model_inputs["input_ids"] = reference_model_inputs.pop("decoder_input_ids")
+        reference_model_inputs["encoder_hidden_states"] = reference_model_inputs.pop("encoder_outputs")[0]
+
+        return reference_model_inputs
+
+    @property
+    def values_override(self) -> Optional[Mapping[str, Any]]:
+        if hasattr(self._config, "use_cache"):
+            return {"use_cache": True}
+
+        return None
+
+
+class SpeechSeq2SeqEncoderOnnxConfig(OnnxConfig):
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("input_features", {0: "batch", 1: "feature_size", 2: "encoder_sequence"}),
```

Comment:
While reviewing Michael's ONNX PR, I realised we can take the opportunity to align our axis names with the same ones used in the `transformers` docs:

```suggestion
                ("input_features", {0: "batch_size", 1: "feature_size", 2: "encoder_sequence_length"}),
```

Reply:
Updated